lib/jxl/modular/encoding/encoding.cc

```diff
@@ -578,9 +589,15 @@ Status ModularDecode(BitReader *br, Image &image, GroupHeader &header,
          channel.h > options->max_chan_size)) {
       break;
     }
-    JXL_RETURN_IF_ERROR(DecodeModularChannelMAANS(
-        br, &reader, *context_map, *tree, header.wp_header, next_channel,
-        group_id, &image));
+    if (code->lz77.enabled)
```

Comment:
braces please!

Reply:
oops, I keep thinking that clang-format will automatically add those but it doesn't...